frontend/components/AddHostsModal/PlatformWrapper/PlatformWrapper.tsx

```diff
@@ -173,7 +178,9 @@ const PlatformWrapper = ({
   const renderInstallerString = (platform: string) => {
     return platform === "advanced"
       ? "osqueryd --flagfile=flagfile.txt --verbose"
-      : `fleetctl package --type=${platform} --fleet-url=${config?.server_url} --enroll-secret=${enrollSecret}`;
+      : `fleetctl package --type=${platform} ${
+          includeFleetDesktop ? "--fleet-desktop " : ""
```

Comment:
@lukeheath 

I just changed:
`${includeFleetDesktop && "--fleet-desktop"}`
to:
`${ includeFleetDesktop ? "--fleet-desktop " : ""}`

because it was rendering the word "false" without the ternary (and I fixed the 1 extra space)

Looks like GitHub is requiring a second review.